src/components/Content/Content.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from "react";
+import { ReactElement, useState } from "react";
+import { ContentProps, ContentType } from "./ContentTypes";
+
+// Components
+import Button from "../Button";
+import Chip from "../Chip";
+import {
+	MainWrapper,
+	Title,
+	TitleWrapper,
+	ButtonsWrapper,
+	Label,
+	Value,
+	LabelValueWrapper,
+	Paragraph,
+	ChipsWrapper,
+	LabelWrapper,
+	ValueFile,
+	ContentRow,
+	ContentColumn,
+	ParagraphWrapper,
+} from "./Content.styled";
+import EditIcon from "@mui/icons-material/Edit";
+import AddIcon from "@mui/icons-material/Add";
+
+const Content = (props: ContentProps): ReactElement => {
+	const { title, onClickEdit, content, onClickAdd } = props;
+	const [showOrHide, setShowOrHide] = useState(content.length < 3);
+
+	/**
+	 * Toggles the state use to show or hide the content.
+	 */
+	const showOrHideDetails = () => {
+		setShowOrHide(!showOrHide);
+	};
+
+	const contentForDisplay = showOrHide ? content : content.slice(0, 3);
+
+	/**
+	 * Renders the content of type "labelValue"
+	 * @param contentItem 
+	 * @returns The content with a sctructure of label: value
+	 */
+	const labelValueContent = (contentItem: ContentType) => (
+		<LabelValueWrapper>
+			<Label>{contentItem.label}:</Label>
+			<Value>{contentItem.value}</Value>
+		</LabelValueWrapper>
+	);
+
+	/**
+	 * Renders the content of type "paragraph"
+	 * @param contentItem 
+	 * @returns A paragraph with the value of the content
+	 */
+	const paragraphContent = (contentItem: ContentType) => (
+		<ParagraphWrapper>
+			<Label>{contentItem.label}:</Label>
+			<Paragraph>{contentItem.value}</Paragraph>
```

Comment:
I believe this is the only thing that changes between the labelValueContent and the paragraphContent, so you could reduce the repeated code by just creating a single element and passing a "type" prop to the styled component and that's where you decide what margin and font-size you apply.

Reply:
This cannot be done because of what I explained in a previous comment, they are different HTML elements.